docs/user/release_notes.rst

```diff
@@ -1129,58 +1191,3 @@ The following contributors submitted patches for Bodhi 2.4.0:
 * andreas
 * Ankit Raj Ojha
 * Randy Barlow
-
-
-2.3.3
```

Comment:
Removing this entire section was intended?

Reply:
Yeah I want to trim the changelog after a year so it doesn't grow without bound. I do similar things in my RPM spec files.

Reply:
I like keeping it, even if it grows since it allows going back to look for when a feature was added or a bug fixed, but up to you :)

My 👍 stands